geode-book/README.md

```diff
@@ -53,7 +53,12 @@ For Geode, a preconfigured **book** is provided in the directory `{geode-project
     $ bundle install
     ```
 
-   Note: You will not have to run `bundle install` on subsequent builds.
+   Note: You will not have to run `bundle install` on subsequent builds. If you see errors during this step regarding the inability to install libv8 try running the command:
+    ```
+    $ gem install libv8 -v '3.16.14.15' --  --with-system-v8
+    ```
+   This can help you avoid errors with the default GCC compiler provided by xcode.
```

Comment:
Xcode instead of xcode. And the solution mentioned is only for MacOS?

Reply:
I only ran this process on MacOS, so this workaround has only been tested on MacOS. The workaround avoids installing libv8 entirely (which is where gcc comes in) so as far as I know this would work on other unix operating systems as well. Having said that, I don't know if other OS would have this problem in the first place as I didn't run this setup on anything else. I will add this info to the document.